Update index.adoc

jnweiger opened the pull request:
Possible fix for https://github.com/owncloud/core/issues/38074

OWNCLOUD_DOMAIN can have a port number, and should have one, if non-standard.
OWNCLOUD_DOMAIN must not have a protocol, though. 

Not sure if OWNCLOUD_PORT should be implicitly applied to OWNCLOUD_DOMAIN  too, aparently it is not.
That would decide, if this an issue with docu or with code.

DeepDiver1975 commented on modules/admin_manual/pages/installation/docker/index.adoc:
```diff
@@ -3,19 +3,23 @@
-This official image works standalone (e.g. for a quick evaluation with `docker run -p{std-port-http}:{std-port-http} owncloud/server`)
-but it is designed to work with a data volume in
-the host filesystem and with separate _MariaDB_ and _Redis_ containers.
+This official image works standalone for a quick evaluation but is designed to be used in a docker-compose setup.
```
no idea why you are changing this .... let people run a stand alone container. What is the problem?

jnweiger replied:
Problem was without `-e DOCKER_DOMAIN=loclahost:8080` it would not fully work. -- I wanted to add that to make it easier to run the standlone container.

DeepDiver1975 replied:
Well - then let's just add this argument and we are fine. No need to change the meaning of the rest of the docs.

jnweiger replied:
The meaning of the rest is unchanged. I removed duplicated stuff and disentangled the language. I fail to see where this is bad. I can split it in two PRs if needed.

DeepDiver1975 replied:
plz split it up. THX

jnweiger replied:
Done. split into #2858 and #2859

DeepDiver1975 reviewed (changes requested):
From my understanding the usage of the port number is already documented.
Adjustments to the docker compose file look valid.

mmattel commented:
Please run things always against master and then do backports. Not the other way round...
Note in the PR descriptipon to which branches you want to backport to.
Exept fixes to a very particular branch, you must not do differnt to avoid possible divergences.
The mess ceaning up is awful...

DeepDiver1975 commented:
.... and a clean commit history with proper commit message .... well I stopped telling people how to work properly ..... :man_shrugging:

jnweiger commented:
split into #2858 and #2859 
added backport hints there. (only the obvious, I am no expert)
closing here unmerged. Hope stuff is cleaner then.

Sorry for being so sloppy with the docs, I am too much priority driven these days.
